app/components/header-nav.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
```

Comment:
New HeaderNav component. Pretty simple, just a container with padding and an explicitly-set z-index